tendrl/gluster_integration/objects/tendrl_context/__init__.py

```diff
@@ -26,7 +26,7 @@ def create_local_integration_id(self):
         with open(tendrl_context_path, 'wb+') as f:
```

Comment:
If there is no way to load `tendrl_context` from local, there is no way we get the details after re-start of `tendrl-gluster-integration` and as mentioned above all the cluster sync details would be wrong.
So effectively we still need loading of `tendrl_context` from local for re-start scenario.
@r0h4n if you any other suggestion, plz provide details here.